src/requests-and-replies.md

```diff
@@ -134,6 +143,8 @@ If two request messages with the same ID are received in short succession (as de
 reply cooldown [parameter](./parameters.md)), the second message should be assumed to be a copy
 sent at the same time as the first, rather than a retransmission, and should be ignored.
 
+// TODO even if both contains different data?
```

Comment:
Yes, as I didn't want to keep the request data around. If you send two messages with the same ID but different contents you might get weird answers but that is your own fault!

Reply:
true (I did not think about it this way :)